types/moko/LW001-BG-PRO/movement.schema.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$id": "https://akenza.io/moko/lw001-bg-pro/movement.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "processingType": "uplink_decoder",
+  "topic": "movement",
+  "title": "Movement",
+  "properties": {
+    "eventType": {
```

Comment:
does it make sense to have a topic for just event type or does this signal an actual movement, if yes, maybe add a movementDetected: true flag